example.v0.tex

```diff
@@ -0,0 +1,95 @@
+\documentclass{refsheet}
+
+% Taken from https://tex.stackexchange.com/a/99784.
+
+% Title area
+\usepackage{titling} % Allows for use of date, author, etc. after \maketitle
+% Ref: http://tex.stackexchange.com/questions/3988/titlesec-versus-titling-mangling-thetitle
+\let\oldtitle\title
+\renewcommand{\title}[1]{\oldtitle{#1}\newcommand{\mythetitle}{#1}}
+\renewcommand{\maketitle}{%
+{\begin{center}\Large \mythetitle\end{center}}
```

Comment:
Why the newline?

Reply:
I don't know. This is the example from the Q&A as it was written there. I decided to keep it. I also don't see why any of that would be necessary, I already redefined the title in the class.